make/RunTests.gmk

```diff
@@ -856,6 +856,11 @@ define SetupRunJtregTestBody
     JTREG_AUTO_PROBLEM_LISTS += ProblemList-zgc.txt
   endif
 
+  ifneq ($$(DEBUG_LEVEL), release)
+    JTREG_AUTO_PROBLEM_LISTS += ProblemList-debug.txt
```

Comment:
We (Oracle) typically do not need this because of the way we run our tests. Hotspot tests are always run on fastdebug in lower tiers and only some tests run on release bits in higher tiers. So the fact the ProblemList affects fastdebug and release is not really an issue. Conversely, JDK tests are run on release bits in lower tiers, and only in some cases run on fastdebug in later tiers.

My main concern here is that we end up with too many potential ProblemList files that have to be managed and maintained. And, IMO jtreg should be enhanced so that the PL syntax allows more flexibility over the conditions under which a test is to be excluded.

Reply:
Thanks for your detail explanation, your concern about too many potential Problemlist files to maintained is reasonable. If no one else support this change, I wll close this PR and the releated issue as `Won't Fix`